GliaWidgets/Sources/Extensions/UIFont+Extensions.swift

```diff
@@ -50,3 +50,22 @@ extension UIFont {
         return UIFont(descriptor: descriptor, size: size)
     }
 }
+
+extension UIFont {
+    static func scaledFont(forTextStyle: UIFont.TextStyle) -> UIFont? {
```

Comment:
Why introduction of methods was necessary? Can you add more comments what problem it solves?

Reply:
Also it seems this change affected other snapshot tests.

Reply:
Added comment here https://github.com/salemove/ios-sdk-widgets/pull/1089/commits/ed646f907a13c766eb024c9b27b5079088269d6f 

Also I commented in snapshot tests about it